src/style.css

```
@@ -0,0 +1,78 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf") format("opentype"),
+    url("./fonts/Avenir-Heavy.ttf") format("opentype");
+}
+
+html {
+  font-weight: 300px;
+  font-family: "Avenir", serif;
+}
+
+.img_big {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.img_small {
+  position: absolute;
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.forms {
+  position: relative;
+  margin: 20px 0;
+  width: 100%;
+}
+
+.big {
+  font-family: inherit;
+  font-size: 16px;
+  padding-left: 62px;
+  height: 70px;
+  line-height: 22px;
+}
+
+.small {
+  font-family: inherit;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: normal;
+  padding-left: 33px;
+  height: 42px;
+  line-height: 19px;
+}
+
+.find {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  color: #3d4e61;
+}
+
+.find:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  box-sizing: border-box;
+}
+
+.find:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
```

```suggestion
  border-color: #e1e7ed;
```
You change only border color here, no need to change the whole border